### PR TITLE
Gb/rpm debug

### PR DESCRIPTION
**Pull request by grantbuster**

### CODE REVIEW

File: reVX/rpm/rpm_output.py [hunk 1]
```diff
@@ -532,23 +558,24 @@ def _get_excl_data(excl, lat_slice, lon_slice):
...
         else:
             e = 'Cannot recognize exclusion type: {}'.format(type(excl))
             logger.error(e)
             raise TypeError(e)
 
         # infer exclusions that are scaled percentages from 0 to 100
-        if excl_data.max() > 1:
-            excl_data = excl_data.astype(np.float32)
-            excl_data /= 100
+        if incl_data.max() > 1:
```
Comment (MRossol) [inline reVX/rpm/rpm_output.py:574]: Start the train! Exclusions make no sense in hindsight!

File: reVX/rpm/rpm_output.py [hunk 2]
```diff
@@ -712,44 +741,58 @@ def _single_excl(cls, cluster_id, clusters, excl_fpath, excl_dict,
...
+            msg = ('Techmap subset must be 1D but received shape {}'
+                   .format(techmap_subset.shape))
+            assert len(techmap_subset.shape) == 1, msg
+            assert len(incl_mask_subset.shape) == 1, msg
 
-        for i, ind in enumerate(clusters.loc[mask, :].index.values):
-            techmap_locs = np.where(
-                techmap == int(clusters.loc[ind, 'gid']))[0]
-            gid_excl_data = exclusions[techmap_locs]
+        inclusions = sum_labels(input=incl_mask_subset, labels=techmap_subset,
```
Comment (MRossol) [inline reVX/rpm/rpm_output.py:787]: So clean!

**Review by MRossol — COMMENTED** [review]
Way to use $$ to do a very much needed refactor!

**Review by MRossol — APPROVED** [review]

File: reVX/rpm/rpm_output.py (hunk 2 above)
Comment (grantbuster) [reply reVX/rpm/rpm_output.py:787]: soooo much faster, great advice!!

File: reVX/rpm/rpm_output.py (hunk 1 above)
Comment (grantbuster) [reply reVX/rpm/rpm_output.py:574]: you mean incl vs. excl? Agreed.... when a method is called "something_exclusions" and returns a variable called "inclusions" you know something has been named poorly...

**Pull request merged**